analyses/snv-callers/README.md

```diff
@@ -1,213 +1,75 @@
-                                  "all_callers_tmb.<file_format>"
-                                  "mutation_id_list.<file_format>"
- --vaf : What VAF should be used when combining callers? Options are 'median' or
-         one of the caller names."
- --combo : What combination of callers need to detect a mutation for it to be
-           considered real and placed in the consensus file? List the callers
-           that need to be considered in alphabetical order with '-'
-           in between. eg. 'lancet-mutect2-strelka2'
- --file_format: What type of file format were the vaf and tmb files saved as? Options are
-               "rds" or "tsv". Default is "rds".
- --output : Where you would like the output from this script to be stored.
- --bed_wgs : File path that specifies the caller-specific BED regions file.
-             Assumes from top directory, 'OpenPBTA-analysis'.
- --bed_wxs : File path that specifies the WXS BED regions file. Assumes file path
-             is given from top directory of 'OpenPBTA-analysis'
- --overwrite : If TRUE, will overwrite any reports of the same name. Default is
-             FALSE
-```
-
-# Individual Caller Evaluation
-
-The first step in this analysis is an individual evaluation of each MAF from each caller.
-This analysis prints out a report that is further subdivided by the WGS and WXS samples.
+It is "MAF-like" file because it has many of the same fields as a MAF file but..  
+  - Does not contain the version string in the first row   
+  - Extraneous annotation data has been removed (columns with all `NA`s)  
+  - Has VAF calculations
 
-Overall reports for each caller and strategy can be found:
-* `results/<caller_name>/<caller_name>_wgs_report.html`
-* `results/<caller_name>/<caller_name>_ws_report.html`
+- `consensus_mutation_tmb.tsv` - After the consensus mutations were identified, Tumor mutation burden was recalculated for each sample from this mutation set.
+See this [analysis' folder](#tumor-mutation-burden-calculation) for details on these methods.
 
-### Base change analysis
+## Summary of Methods
 
-To evaluate base changes I summarized standard MAF fields as two new variables:
-The `base_change` variable that indicates the exact change in bases from
-concatenating `Reference_Allele`, `>`, and `Allele`.
-The `change` variable is made from the `base_change` variable but groups
-together deletions, insertions, and long (more than a SNV) as their own groups.
+### Mutation Comparisons
 
-*Output for this analysis*
-* `results/<caller_name>/<caller_name>_vaf.rds`
-* `plots/<caller_name>/<caller_name>_<strategy>_base_change.png`
+In order to compare mutations across callers, the data tables for each caller
+were indexed by: `Chromosome` `Start_Position` `Reference_Allele` and `Allele`.
+Meaning that across callers, if all these fields were identical, they were considered to be the same mutation.
+This was done in the `01-setup_db.py` script using the function.
```

Comment:
The indexing is an implementation detail for speed, it speeds comparisons later, which also end up including sample for matching (but we get enough speed without specifically indexing that as well).

So I think this section needs to just describe which callers were used for the consensus, what was considered the same mutation (the fields mentioned, but also sample).  and how we handle MNVs.

Reply:
Would you be able to add a few lines about how you did the MNVs? Or do you have elsewhere where that is summed up nicely?

Reply:
I will add it. I think the best summary is in #275, but it isn't quite as clear as it could be. I'll write something better tomorrow morning.

Reply:
Sounds good. You can always add it as a PR suggestion and then I can just commit it.